packages/ui-text/src/Text/README.md

```diff
@@ -4,6 +4,28 @@ describes: Text
 
 A component for styling textual content
 
+## Variant
+
+Variant takes care of - almost - all use-cases when it comes to texts on pages. Their name reflects the places they meant to be used. It sets `size`, `weight`, `fontStyle` and `lineHeight`
+We recommend using `variants` instead of the aforementioned props.
+
+NOTE: when `variant` is set, `size`, `weight`, `fontStyle` and `lineHeight` props are ignored
```

Comment:
Why is this behaving differently from Heading? We are there not ignoring the values set by `variant`, just logging a warning. IMO we should be consistent, I vote for variant overwriting these props

Reply:
They work the same way. When variant is set, every other prop that effect the same things as the variant is being ignored or overwritten. Same thing in this case. The warning is for cleaning up the users code